types/karma-fixture/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for karma-fixture 0.2.7
+// Type definitions for karma-fixture 0.2.6
```

Comment:
As part of this PR, can you also:
- remove the patch version from the version number (so it just reads `karma-fixture 0.2`)
- remove the line in the `tslint.json` in this folder that is disabling the `dtheader` rule.

It would be great if you could remove all the rule disabling in the `tslint.json`, but that might be a lot of work.

Reply:
I may give it a try.

Reply:
Let me know if you have any questions.